WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css

```diff
@@ -193,4 +193,32 @@ h3 {
 
 .detail-block-1, .detail-block-2 {
 	display: block;
-}
\ No newline at end of file
+}
+
+#container {
+	opacity: 1;
+}
+
+#overlay {
+	position: fixed;
```

Comment:
no need for 8 spaces here, 4 spaces for the indentation should be enough

Reply:
Lines like line 203 indented by  1 tab (4spaces). Could you elaborate?

Reply:
Then that's the preview on github. good